### PR TITLE
ODC-7417: Add scripts for Development setup

### PR DESCRIPTION
**Pull request by saumeya**
This PR adds a scripts for installing the operator and results on the cluster, the user can specify the version of the operator like so :- 
./install-operator-results-api.sh 1.13.1

https://issues.redhat.com/browse/ODC-7417

### CODE REVIEW

File: scripts/install-operator-results-api.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  channel: pipelines-1.12
+  config:
+    env:
+      - name: IMAGE_RESULTS_API
+        value: quay.io/avinkuma/api-b1b7ffa9ba32f7c3020c3b68830b30a8@sha256:e1b184582f9c42fa5493edab03a76ae3adc61359217728028bf11502e144dfd2
+  installPlanApproval: Automatic
+  name: openshift-pipelines-operator-rh
+  source: custom-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: openshift-pipelines-operator-rh.v1.12.2
```
Comment (khrm) [inline scripts/install-operator-results-api.sh:28]: Change this to `openshift-pipelines-operator-rh.v1.13.0`

File: scripts/install-operator-results-api.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+cat << EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/openshift-pipelines-operator-rh.openshift-operators: ""
+  name: openshift-pipelines-operator-rh
+  namespace: openshift-operators
+spec:
+  channel: pipelines-1.12
```
Comment (khrm) [inline scripts/install-operator-results-api.sh:19]: Change this to `pipelines-1.13`

File: scripts/install-operator-results-api.sh (hunk 1 above)
Comment (vdemeester) [reply scripts/install-operator-results-api.sh:28]: Not that it shouldn't be required (`startingCSV` is optional)

File: scripts/install-operator-results-api.sh [hunk 3]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script assumes that you are connected to an OpenShift cluster and have the Pipelines Operator installed
+
+echo "Installing the Pipelines Operator ... "
```
Comment (vdemeester) [inline scripts/install-operator-results-api.sh:7]: To give an example, this is how, on ecosystem, we install the operator : https://github.com/openshift-pipelines/task-containers/blob/main/hack/install-osp.sh

File: scripts/install-operator-results-api.sh (hunk 2 above)
Comment (saumeya) [reply scripts/install-operator-results-api.sh]: made it latest

**Comment by jerolimov** [comment]
/retest

@vikram-raj @saumeya Should we merge this and then improve step by step?

@saumeya Is Avinals tekton results image sha still valid or can we use there latest as well?

File: scripts/install-operator-results-api.sh [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  labels:
+    operators.coreos.com/openshift-pipelines-operator-rh.openshift-operators: ""
+  name: openshift-pipelines-operator-rh
+  namespace: openshift-operators
+spec:
+  channel: latest
+  config:
+    env:
+      - name: IMAGE_RESULTS_API
+        value: quay.io/avinkuma/api-b1b7ffa9ba32f7c3020c3b68830b30a8@sha256:e1b184582f9c42fa5493edab03a76ae3adc61359217728028bf11502e144dfd2
```
Comment (vikram-raj) [inline scripts/install-operator-results-api.sh:23]: Update this value to `quay.io/avinkuma/api-b1b7ffa9ba32f7c3020c3b68830b30a8:latest`

**Comment by saumeya** [comment]
> /retest
> 
> @vikram-raj @saumeya Should we merge this and then improve step by step?
> 
> @saumeya Is Avinals tekton results image sha still valid or can we use there latest as well?

Sure, @jerolimov  - i'll remove the autocreate pipelies part as we have your script for test data. I will update the image tag

**Review by vikram-raj — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**